compare_mt/compare_mt_main.py

```diff
@@ -364,6 +367,44 @@ def generate_sentence_examples(ref, outs, src=None,
   reporter.generate_report()
   return reporter 
 
+def generate_lang_id_report(ref, outs, model,
+                            min_length=5,
+                            print_lines=False,
+                            print_line_numbers=False):
+    wtl = WhatTheLang()
+    lang_id_reports=[]
+    lang_id_lines_reports=[]
+    lang_id_line_numbers_reports=[]
+    for out in outs:
+        langs = defaultdict(int)
+        lang_lines = defaultdict(list)
+        lang_line_numbers = defaultdict(list)
+        for i, sentence in enumerate(out, start=1):
+            line = corpus_utils.list2str(sentence)
+            if len(sentence) >= int(min_length):
+                if model=="langid":
+                    (lang, prob) = langid.classify(line)
+                elif model=="wtl":
```

Comment:
If `model` is neither "wtl" nor "langid", this should be caught somewhere. For instance, with a NotImplementedError:

```python
if model=="langid":
    (lang, prob) = langid.classify(line)
elif model=="wtl":
    lang = wtl.predict_lang(line)
else:
    raise NotImplementedError(f"Unknown model for language identification: '{model}'.")
```

Reply:
done